src/cmd/services/m3aggregator/config/aggregator.go

```diff
@@ -89,7 +89,7 @@ type AggregatorConfiguration struct {
 	GaugePrefix *string `yaml:"gaugePrefix"`
 
 	// Stream configuration for computing quantiles.
-	Stream streamConfiguration `yaml:"stream"`
+	Stream *streamConfiguration `yaml:"stream"`
```

Comment:
Why make this nillable?

Reply:
I was trying to cut the unused part in the test config yaml.
Just changed it back and added a minimal stream configuration in the test.